lib/buy/order.dart

```diff
@@ -42,24 +42,24 @@ class Order extends HiveObject {
   @HiveField(3)
   String? to;
 
-  @HiveField(4)
+  @HiveField(4, defaultValue: '')
   late String stateRaw;
```

Comment:
with the current implementation of the constructor, this variable may not be initialized if the state is null, resulting in a `late variable not initialized` exception